hooks/migrate_sdsdrbd_crds.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import kubernetes
+from deckhouse import hook
+import time
+
+from re import search
+from time import sleep
+
+config = """
+configVersion: v1
+afterHelm: 10
+"""
+
+def main(ctx: hook.Context):
+    kubernetes.config.load_incluster_config()
+
+    # Delete old webhooks
+    custom_object = kubernetes.client.CustomObjectsApi().list_cluster_custom_object(group='admissionregistration.k8s.io',
```

Comment:
Why don't helm deletes webhooks?

Reply:
It is some bug in d8, after deleting module, not every resource is being deleted.